lib/utils/logging.js

```diff
@@ -0,0 +1,155 @@
+import EventEmitter from 'events'
+
+import bfj from 'bfj-node4'
+import { isObject } from 'lodash'
+import figures from 'figures'
+import moment from 'moment'
+
+import getEntityName from './get-entity-name'
+
+export const logEmitter = new EventEmitter()
+
+function extractErrorInformation (error) {
+  const data = JSON.parse(error.message)
```

Comment:
isn't this try & catch enough?
https://github.com/contentful/contentful-batch-libs/pull/126/files/09b5783d4d3dc5dcb0656cab4026e5cce423ee3b#diff-e515acbb2b685b01df89f42b247f8d11R34